frappe/www/login.py

```diff
@@ -220,3 +221,24 @@ def sanitize_redirect(redirect: str | None) -> str | None:
 			output_parsed_url = output_parsed_url._replace(path=parsed_redirect.path)
 
 	return output_parsed_url.geturl()
+
+
+@frappe.whitelist(allow_guest=True)
+@rate_limit(key="mobile_no", limit=5, seconds=60 * 10)
+def send_mobile_otp(mobile_no: str | None) -> None:
+	from frappe.utils.mobile_otp import find_user_by_mobile, send_mobile_login_otp
+
+	if not mobile_no:
+		frappe.throw(_("Mobile number is required"))
```

Comment:
```suggestion
```

Explicit check not required - you should get `TypeError: send_mobile_otp() missing 1 required positional argument: 'mobile_no'`